dask/dataframe/io/parquet/core.py

```diff
@@ -347,7 +254,21 @@ def read_parquet_part(func, fs, meta, part, columns, index, kwargs):
     """ Read a part of a parquet dataset
 
     This function is used by `read_parquet`."""
-    df = func(fs, part, columns, index, **kwargs)
+    if isinstance(part, list):
+        concat_func = kwargs.pop("concat_func", False)
+        if not concat_func:
+            import pandas as pd
+
+            concat_func = pd.concat
```

Comment:
Is this `concat_func` keyword used?  

Instead, I recommend that we use `dask.dataframe.methods.concat`